lmod/admin.list

```diff
@@ -47,6 +47,27 @@ python/3.5.2: 		This module is deprecated and will be removed in the future. Ple
 			Ce module est marqué pour être retiré et sera supprimé dans le futur. Veuillez utiliser python/3.5. Si vous aviez créé un environnement
 			virtuel, nous vous recommandons de le recréer.
 
+python/3.6.10:          Support for Python 3.6 has ended in Decembre 2021. Please upgrade to the latest version of Python. 
+                        Le soutien de Python 3.6 s'est arrêté au mois de décembre 2021. Veuillez mettre à jour votre travail vers la version la plus récente.
```

Comment:
Hmm no, the latest version of Python is a moving target, it's 3.10.5 for which we don't have a module, and it'll soon mean 3.11! That's not what I meant.

Perhaps "3.8 or later" is better.

Reply:
But when we install Python 3.11, we will probably stop building for Python 3.8, so that will be a moving target too ? For maximum longevity, it is best if users upgrade to the latest version that we have available.

Reply:
I like the messages for python/3.7 better, as they clearly state which versions we support.

Maybe the sentence:
> Please upgrade your workflow to the latest version of python.
Could be replaced with:
> We recommend that you upgrade your workflow to the latest version of python.

Reply:
The problem with the latest version of Python is often that it is too new, it takes a while for the ecosystem to catch up.
E.g. when 3.11 comes out later this year many packages won't work with it straightaway.
In a sense the default version (still 3.8 at the moment) is an ok target.

Reply:
FYI: Gentoo Linux (i.e., a rolling distribution built from source so can afford to be more liberal and quick to upgrade) only recently (i.e., July 1, 2022) has gone stable (as opposed to "testing") for Python 3.10... so that implies newly released code for a significant number of current softwares have *just* caught up (e.g., no build failures, etc.) to Python 3.10. IMO Python 3.9 should be safe as a main minimal version as a default.

Reply:
I reworded